openwisp_monitoring/device/admin.py

```diff
@@ -145,6 +160,20 @@ def get_form(self, request, obj=None, **kwargs):
         )
         return super().get_form(request, obj, **kwargs)
 
+    def get_fields(self, request, obj=None):
+        if not obj or obj.monitoring.status != 'problem':
```

Comment:
I think we should show this also when the status is critical, it's helpful, moreover the system can be configured to have other metrics which are critical and may trigger this status.